frontend/src/metabase/admin/performance/hooks/useVerticallyOverflows.tsx

```diff
@@ -0,0 +1,13 @@
+import { useEffect, useRef, useState } from "react";
+
+export const useVerticallyOverflows = () => {
+  const [verticallyOverflows, setVerticallyOverflows] = useState(false);
+  const ref = useRef<HTMLElement>(null);
+  const el = ref.current;
+  useEffect(() => {
+    if (el) {
+      setVerticallyOverflows(el.scrollHeight > el.clientHeight);
+    }
+  }, [el?.scrollHeight, el?.clientHeight, el, setVerticallyOverflows]);
+  return { verticallyOverflows, ref };
+};
```

Comment:
@sloansparger The variable should update when the scrollHeight or clientHeight changes - that's why it requires `useLayoutEffect`